ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/attestation/ForkChoiceAttestationProcessor.java

```diff
@@ -33,7 +33,7 @@ public ForkChoiceAttestationProcessor(
   public AttestationProcessingResult processAttestation(final Attestation attestation) {
     final Store.Transaction transaction = recentChainData.startStoreTransaction();
     final AttestationProcessingResult result = forkChoice.onAttestation(transaction, attestation);
-    if (result.isSuccessful()) {
+    if (result == AttestationProcessingResult.SUCCESSFUL) {
```

Comment:
nit: Would probably good to add the `isSuccessful()` method to the enum as it reads quite well.

Reply:
done.